### PR TITLE
Debsync20191206 python refinements

### PR DESCRIPTION
**Pull request by bazsi**
This PR is against #4 so that should go in first. This branch concentrates on Python related changes, that are:

* it adds older names for various Python packages so that the packaging supports older Ubuntu/Debian releases. I felt these were OK in the upstream, otherwise we would have to maintain them.

* it moves the Python source code into a syslog-ng specific prefix (e.g. /usr/lib/syslog-ng/python) to follow upstream. These files don't really make sense outside of syslog-ng (e.g. they rely on Python functions exported by the syslog-ng core, so they wouldn't work anywhere else). I can only vaguely remember the exact reasoning in upstream, but I can look that up (or involve @MrAnno) who was behind the changes.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -26,9 +26,9 @@ Build-Depends: debhelper (>= 10~),
                libhiredis-dev,
                libriemann-client-dev (>= 1.6.0~),
                librabbitmq-dev,
-               python3, python3-dev, pep8, pylint3, dh-python,
+               python3, python3-dev, python3-pep8 | pep8, pylint3, dh-python,
```
Comment (mochrul) [inline debian/control:29]: I'm not sure I understood what's going here, but if you added the python3-pep8 for backward-compatibility, I would suggest removing pep8, as pep8 is currently just a transitional package which does nothing else than depending on python3-pep8.

Comment (bazsi) [reply debian/control:29]: This was originally converted from python2 equivalents, and Debian jessie didn't have a python-pep8 package (but it did have pep8). Now it does have python3-pep8, so this is indeed not needed.

Comment (bazsi) [reply debian/control:29]: I am dropping this patch if we can merge #3 so I can rebase the rest of the patches on top of that.

**Comment by MrAnno** [comment]
Hello,

@mochrul @bazsi Sorry for making noise in this PR, I would like to ask your opinion about something I was working on:

I have a pull request that makes it possible to build syslog-ng with both the Python 2 and Python 3 modules enabled (syslog-ng/syslog-ng#2845).

This means that it would be possible to package both modules separately, and the user could choose between versions. I know about the Python 2 EOL, but this may help users migrate to Python 3, because they could quickly switch between versions using the `python3()` and `python()`/`python2()` keywords in the config. Both modules can't be started within a single syslog-ng process due to conflicting symbols, but we have proper runtime detection for this.

The patch is not trivial, so we don't want to merge it unless someone finds it useful.  Is this something you could/would/want to use in the syslog-ng Debian packaging, or is it too late? :)

**Comment by bazsi** [comment]
I am doing another sync round, so this has become obsolete. Closing.

@MrAnno's question is still valid in this PR though.

**Pull request closed without merging**